stable_baselines/common/policies.py

```diff
@@ -235,40 +232,113 @@ class LstmPolicy(ActorCriticPolicy):
     :param n_lstm: (int) The number of LSTM cells (for recurrent policies)
     :param reuse: (bool) If the policy is reusable or not
     :param layers: ([int]) The size of the Neural network before the LSTM layer  (if None, default to [64, 64])
+    :param net_arch: (list) Specification of the actor-critic policy network architecture. Notation similar to the
+    format described in mlp_extractor but with additional support for a 'lstm' entry in the shared network part.
```

Comment:
```suggestion
        format described in mlp_extractor but with additional support for a 'lstm' entry in the shared network part.
```
Minor: indent comment, otherwise sphinx complains:

```
stable_baselines/common/policies.py:docstring of stable_baselines.common.policies.LstmPolicy:13:
WARNING: Field list ends without a blank line; unexpected unindent.
```